chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/SurveillanceValidator.java

```diff
@@ -349,4 +355,26 @@ public void validateSurveillanceNonconformities(Surveillance surv) {
 			}
 		}
 	}
+	
+	public void validateSurveillanceAuthority(Surveillance surv) {
```

Comment:
#1. Agreed that the Surveillance object's authority will only have one String value. Regarding the return of Unauthorized - we're doing authorization in the managers with the @PreAuthorize annotation. The purpose of the SurveillanceValidator is primarily to add an error message if the business logic is not met. I don't think we should use the value of the Surveillance object's authority to determine authorization; I think we should use the User object's role and permissions on the ACB. Currently, we _will_ return unauthorized because of the manager methods' @Preauthorize annotations. I thought that an error message would suffice for the case when an ACB_ADMIN tries to update a surveillance that was created by ROLE_ADMIN, but if we're in agreement that this should instead cause an "Unauthorized" error, then I'll go ahead and change that functionality.
#2. I think I misinterpreted the requirement in OCD-1180 for "ONC Admin" and OCD-1250 for "ONC user" as being ROLE_ONC_STAFF. I think I need to change references to ROLE_ONC_STAFF to ROLE_ADMIN. I think I also need to add the ability for surveillance authority to equal ROLE_ACB_STAFF, right? Or will you always pass in ROLE_ACB_ADMIN for ACB users?
#3. See my comment to #2.